src/components/Common/SelectRecordListFields.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div>
+    <b-button @click="showModal=true"
+      size="lg"
+      variant="light"
+    >
+      Select fields
+    </b-button>
+
+    <b-modal :visible="showModal"
+      @hide="onModalHide"
+      size="lg"
+      title="Select fields"
+      :ok-title="$t('general.label.saveAndClose')"
+      body-class="p-0"
+    >
+      <div
+        class="card-body"
+      >
+        <field-picker
+          :module="module"
+          :fields.sync="filteredFields"
+        />
+      </div>
+    </b-modal>
+  </div>
+</template>
+
+<script>
+import FieldPicker from 'corteza-webapp-compose/src/components/Common/Module/FieldPicker'
+
+export default {
+  components: {
+    FieldPicker,
+  },
+
+  props: {
+    module: {
+      type: Object,
+      required: true,
+      default: () => ({}),
+    },
+    fields: {
+      type: Array,
+      required: true,
+      default: () => [],
+    },
+  },
+  data () {
+    return {
+      showModal: false,
+      filteredFields: [],
```

Comment:
filteredFields is empty no matter if fields are selected or not.
So make a watcher on fields prop, and update filteredFields when they change.